Automattic/woocommerce-payments
# Update: woocommerce component version to 6.1.2

Fixes: #1881 

#### Changes proposed in this Pull Request
- Updated woocommerce component version.
- Fixed -ve values being misaligned while exporting CSV.









#### Testing instructions
1. Go to the Transactions page.
2. Make sure you have some `-ve` values, you can do this by creating a refund test transaction.
2. Click on Download.
3. You'll see a negative value represented as-is.

- With this update, let's make sure if other woocommerce components are working fine.
- Change log: https://github.com/woocommerce/woocommerce-admin/blob/main/packages/components/CHANGELOG.md





-------------------
